api/conftest.py

```diff
@@ -966,9 +967,22 @@ def flagsmith_environments_v2_table(dynamodb: DynamoDBServiceResource) -> Table:
 
 
 @pytest.fixture()
-def feature_external_resource(feature: Feature) -> FeatureExternalResource:
+def feature_external_resource(
+    feature: Feature, mocker: MockerFixture
+) -> FeatureExternalResource:
+    from tests.unit.features.test_unit_feature_external_resources_views import (
+        mocked_requests_post,
+    )
```

Comment:
I think `mocked_requests_post` should be converted into a fixture and added to a `conftest.py` instead of importing it like this.

Reply:
Converted into a fixture in `conftest.py`